quodlibet/qltk/quodlibetwindow.py

```diff
@@ -957,6 +958,11 @@ def add_top_level_items(ag):
         act.connect('activate', self.__next_song)
         ag.add_action_with_accel(act, "<Primary>period")
 
+        act = Action(name="Stop", label=_("Stop"),
+                     icon_name=Icons.MEDIA_PLAYBACK_STOP)
+        act.connect('activate', self.__stop)
+        ag.add_action(act)
```

Comment:
**question:** Should we add a keyboard shortcut?

Some suggestions: Ctrl+Shift+Space, Ctrl+Escape

Reply:
Maybe yes. Not Ctrl+Escape I'd say, that's too weird and might do things in some WMs